lib/enum/basic_enum.rb

```diff
@@ -78,6 +89,7 @@ def groups_of(num)
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-each_with_object
   # Tip: You'll have to do .each_with_object(Hash.new(0)) to create a hash to start with. Do not save the hash in a variable. You may use the incrementor method here.
   def element_frequency_count
+    collection.each_with_object(Hash.new(0)) { |num, word| word[num] += 1 }
```

Comment:
each_with_object does take two parameters, but they are NOT `|num, word|`. Instead, they are `|key, new_hash|` where new_hash is the hash that you created with Hash.new(0). So it would not make sense to say `word[num] += 1`, but it does make sense to create new keys with `new_hash[key] += 1`. This particular problem is quite intricate - make sure you understand what you're doing when you write the code, or if you're not sure, write a comment about it.